seqr/views/apis/dataset_api_tests.py

```diff
@@ -296,6 +202,150 @@ def test_add_variants_dataset(self, mock_open, mock_random, mock_redis):
         self.assertSetEqual({sv_sample_guid, existing_index_sample_guid, new_sample_type_sample_guid}, {sample.guid for sample in sample_models})
         self.assertSetEqual({True}, {sample.is_active for sample in sample_models})
 
+    @urllib3_responses.activate
+    def test_add_variants_dataset_errors(self):
+        url = reverse(add_variants_dataset_handler, args=[PROJECT_GUID])
+        self.check_data_manager_login(url)
```

Comment:
I assume the tests for the errors have no change.

Reply:
I added a couple, and the order had to change because its now calling validate on the index metadata before it is validating that there are samples in the index, instead of the other way around. But nothing was removed. I'll comment on the new ones so you can find them